init-tools.sh

```diff
@@ -144,24 +145,22 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
 
         echo "Initializing BuildTools..."
         echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
+
+        chmod +x $__BUILD_TOOLS_PATH/init-tools.sh
         $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR >> $__init_tools_log
         if [ "$?" != "0" ]; then
             echo "ERROR: An error occured when trying to initialize the tools. Please check '$__init_tools_log' for more details."1>&2
             exit 1
         fi
     fi
 
-    if [ $__PATCH_CLI_NUGET_FRAMEWORKS -eq 1 ]; then
-        echo "Updating CLI NuGet Frameworks map..."
-        cp $__TOOLRUNTIME_DIR/NuGet.Frameworks.dll $__TOOLRUNTIME_DIR/dotnetcli/sdk/$__DOTNET_TOOLS_VERSION >> $__init_tools_log
-        if [ "$?" != "0" ]; then
-            echo "ERROR: An error occured when updating Nuget for CLI . Please check '$__init_tools_log' for more details."1>&2
-            exit 1
-        fi
-    fi
-
     Tools-Override/crossgen.sh $__scriptpath/Tools
 
+    echo "Making all .sh files executable under Tools."
+    ls $__scriptpath/Tools/*.sh | xargs chmod +x
+
+    cp Tools-Override/* Tools
```

Comment:
This is currently being done in run.sh https://github.com/dotnet/corefx/blob/master/run.sh#L10.